examples/plotting/lon_lat_lines.py

```diff
@@ -16,38 +17,37 @@
 from sunpy.data.sample import AIA_171_IMAGE
 
 ###############################################################################
-# The purpose of this example is to demonstrate the coordinate transformations
-# that occur under the hood to show the heliographic grid lines of longitude
-# latitude. We first create the Map using the sample data.
+# We will demonstrate the coordinate transformations that occur under the
+# hood to show the heliographic grid lines of longitude latitude.
+# We first create a map using the sample data.
 
 aia = sunpy.map.Map(AIA_171_IMAGE)
 
 ###############################################################################
-# Let's first transform a single heliographic point coordinate.
+# Let's first specify a single heliographic point coordinate.
 
 stonyhurst_center = SkyCoord(12 * u.deg, 12 * u.deg,
                              frame=frames.HeliographicStonyhurst)
 
 ###############################################################################
-# Next we transform it into the coordinate frame of our map which is in
-# helioprojective coordinates.
+# Next we transform it into the coordinate frame of our map which is helioprojective.
 
 hpc_stonyhurst_center = stonyhurst_center.transform_to(aia.coordinate_frame)
 print(hpc_stonyhurst_center)
 
 ###############################################################################
-# Now let's define two lines, one of longitude and one of of latitude.
+# Now let's define two lines, one of constant longitude and one of constant latitude.
 # We don't need to explicitly transform them to the coordinate frame of our
 # map because they will be transformed automatically when plotted.
 
 num_points = 100
 lat_value = 12 * u.deg
 lon_value = 35 * u.deg
-lon0 = SkyCoord(np.linspace(-80, 80, num_points) * u.deg,
-                np.ones(num_points) * lon_value, frame=frames.HeliographicStonyhurst)
-lat0 = SkyCoord(np.ones(num_points) * lat_value,
-                np.linspace(-90, 90, num_points) * u.deg,
-                frame=frames.HeliographicStonyhurst)
+lon = SkyCoord(np.ones(num_points) * lon_value,
+               np.linspace(-90, 90, num_points) * u.deg,
+               frame=frames.HeliographicStonyhurst)
+lat = SkyCoord(np.linspace(-90, 90, num_points) * u.deg,
+               np.ones(num_points) * lat_value, frame=frames.HeliographicStonyhurst)
```

Comment:
```suggestion
constant_lon = SkyCoord(lon_value, np.linspace(-90, 90, num_points) * u.deg,
                        frame=stonyhurst_frame)
constant_lat = SkyCoord(np.linspace(-90, 90, num_points) * u.deg, lat_value,
                        frame=stonyhurst_frame)
```
The single value will automatically be broadcast, so there's no need to explicitly create an array of constant value.